explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -681,19 +689,30 @@ sealed abstract class AdvancedConfigurationPanelBuilder[
               "Binning",
               HelpIcon("configuration/binning.md".refined)
             ),
-            customizableEnumSelectOptional(
-              id = "explicitXBin".refined,
-              view = explicitBinning(props.observingMode).withDefault(defaultBinning),
-              original = defaultBinning.some,
-              disabled = disableAdvancedEdit
+            <.div(
+              ExploreStyles.AdvancedConfigurationBinning,
+              customizableEnumSelectOptional(
+                id = "explicitXBin".refined,
+                view = explicitXBinning(props.observingMode).withDefault(defaultXBinning),
+                original = defaultXBinning.some,
+                disabled = disableAdvancedEdit
+              ),
+              FormLabel(htmlFor = "explicitYBin".refined)("x"),
+              customizableEnumSelectOptional(
```

Comment:
Shouldn't this have a formLabel too?

Reply:
Both the dropdowns do have labels. However, your comment made me realize that the current labels are not very helpful for screen readers, so I disassociated the labels from the dropdowns and used `aria-label` directly on the dropdowns.